### ISSUE TITLE
The configuration assistant is behaving strangely and showing lot of `Gtk-CRITICAL` warnings in the terminal

### ISSUE THREAD

**Issue by peterpalau** [coyim/coyim#762]
Recently, we realized that the configuration assistant has a strange behavior in some cases.

For example, when you interact with the assistant and change the pages continuously, at some point, it starts to show the following critical messages:

```
...
(CoyIM:185950): Gtk-CRITICAL **: 09:33:03.227: gtk_widget_set_visible: assertion 'GTK_IS_WIDGET (widget)' failed
(CoyIM:185950): Gtk-CRITICAL **: 09:33:03.227: gtk_widget_set_visible: assertion 'GTK_IS_WIDGET (widget)' failed
(CoyIM:185950): Gtk-CRITICAL **: 09:33:03.227: gtk_widget_set_visible: assertion 'GTK_IS_WIDGET (widget)' failed
....
```

Maybe this happens because we loop over all the assistant children and tries to cast each of them to a specific `GTK` widget type.

Another thing that is happening is when we disable wizard navigation because the current page has at least one validation error.

We realized that when the user types a password in the password field, and that password is not equal to the confirmation password, the navigation assistant is disabled. But when the user SHOWS the password, the validation error goes away but the navigation buttons are not enabled again.

**Comment by olabiniV2** [comment]
For this kind of issue, it would be a good idea to actually describe in the issue what was happening, so that if in the future we see similar problems we can have a consise description of the solution here.

**Comment by olabiniV2** [comment]
Reviewed, otherwise good

**Comment by peterpalau** [comment]
We tested this issue with @sacurio and looks good. The issue should be reviewed by @olabiniV2.

**Comment by olabiniV2** [comment]
OK

**Comment by olabiniV2** [comment]
It seems that this issue was never actually fixed. It seems to be the same underlying issue as #775 so we will have to re-open it and track it there.

**Comment by olabiniV2** [comment]
This was fixed as part of #775
